### PR TITLE
[READY] [Issue-567] - Adding OpenWRT Grafana Dashboard

### LINKED ISSUE #567: monitoring and make nice cute graphs of network ....stuff
## Description

Monitoring of various network stats need to be configured and deployed.

whats my options.

## Acceptance Criteria

Nice graphs!!!

### PR DESCRIPTION
**Pull request by jshcmpbll**
## Description of PR
relates to: #567 

## Previous Behavior
No dashboards configured by default for grafana

## New Behavior
openwrt grafana dashboard

## Tests
nix build
@sarcasticadmin is going to test with a local OpenWRT deployment since I dont have any hardware

### CODE REVIEW

File: nix/machines/monitor.nix [hunk 1]
```diff
@@ -65,11 +66,18 @@ in
         datasources.settings.datasources = [
           {
             name = "prometheus";
+            uid  = "P1809F7CD0C75ACF3";
```
Comment (kylerisse) [inline nix/machines/monitor.nix:69]: just to clarify, does this value have any specific meaning or does it just need to be $SOME_VALUE that matches all the datasource references form the dashboard json?

Comment (sarcasticadmin) [reply nix/machines/monitor.nix:69]: Im unclear on this myself

@jshcmpbll might be able to add some insight

Comment (jshcmpbll) [reply nix/machines/monitor.nix:69]: @kylerisse Yup! The value itself has no real meaning but would need to match the data source values found in the dashboard like so:

https://github.com/socallinuxexpo/scale-network/blob/9258aee2c9590d4bfe924cad478c6c2b6debbe05/monitoring/openwrt_dashboard.json#L72

Each time I started the VM it seemed to use that UID for the datasource by default but I figured we might as well set it so there isn't a possibility of some change in the future resulting in the UID changing.

**Comment by jshcmpbll** [comment]
> @jshcmpbll This looks like exactly what we need!
> 
> Going to work on getting this connected to openwrt shortly

I started working on the exporter replay test we talked about but still trying to figure out networking for nix tests.

**Review by owendelong — APPROVED** [review]
LGTM

**Comment by owendelong** [comment]
Not merging because marked "REVIEW". If you want it merged, please mark it READY.

File: nix/machines/monitor.nix (hunk 1 above)
Comment (kylerisse) [reply nix/machines/monitor.nix:69]: sweet, thanks @jshcmpbll

**Review by sarcasticadmin — APPROVED** [review]
I was able to setup an AP and run this monitoring vm. The dashboard populates as expected and everything seems to be coming in nicely! (ignore the wifi quality metric, I really didnt gather too much data for this initial test)

ss-202402231708670002

I added a small change in the inventory.py to ensure that we are using `port 9100` for the metrics exporters on the APs.

Nice work @jshcmpbll ! This is a big step for getting the dashboard built out!

**Pull request merged**